docs/explanation/archive-administration/plus-one-maintenance.md

```diff
@@ -2,3 +2,132 @@
+channel on Matrix.
+
+The Debcrafters team, and Simon Chopin in particular, is responsible for
+coordinating the effort.
+
+## The work
+
+### Shift report
+
+Regardless of the exact work, it is advised to keep track of what has been
+accomplished during a shift. This will serve as the basis for a +1 maintenance
+report to be posted to ubuntu-devel@lists.ubuntu.com at the end of shift.
+
+There is no formal structure for the report, but it is advised to frontload the
+report with any information or issue that a subsequent shift might want to know
+about. It could be an investigation where progress was made but needs carrying
+to conclusion, some uploads that would unblock packages but are still waiting
+for sponsorship, an upcoming transition from Debian, etc.
+
+Besides this, some will choose to describe in details some interesting
+investigation they worked on, others will list exhaustively the packages and
+bugs they worked on.
+
+### Coordinating the effort
+
+There are plenty of people working on Ubuntu, and there even can be multiple
+people on +1 at the same time. To avoid stepping on each other's toes,
+communication is crucial.
+
+When working on a package, contributors should first check Launchpad for a
+relevant open bug, and if it has been assigned to someone. If not, they should
+open the bug if needed, and assign it to themselves. It is advised to be
+liberal in documenting one's progress on the bug itself. When opening a new
+bug, by convention, it should be tagged `update-excuse`, since in most cases
+the things being worked on affect the migration of a package.
+
+The tool `pm-helper` can be used for the creation of such bugs.
+
+### Finding work
+
+As mentioned, the focus is the `devel` series. Depending on when the shift
+takes place in the cycle, the tasks can vary. The first thing to check for
+should be the previous shift report for actionable items — see below.
+
+One can also ask on Matrix if there are transitions that people might need help
+for, e.g. a new version of Python.
+
+Failing that, there are a few places that one might want to look at to find
+work. They are loosely listed in order of priority, but that importance varies
+depending on the development cycle.
+
+#### Transition tracker, NBS
+
+The [NBS tracker](https://ubuntu-archive-team.ubuntu.com/nbs.html), for "Not
+Buildable from Sources", tracks binary packages in the archive that cannot be rebuilt from source.
+While that can happen for several reasons, by far the most common is that the binary
+in question is the old version of a library that is having an ABI transition.
+
+A more specialized view for transitions is the [transition
+tracker](https://ubuntu-archive-team.ubuntu.com/transitions/) but it might not
+track all ongoing transitions since it requires manual intervention to setup a
+new tracker.
+
+#### FTBFS
+
+The priority of these rises along with the cycle. We want as few packages failing to build
```

Comment:
suggestion: Add one sentence at the beginning of the paragraph explaining what FTBFS stands for.

Reply:
Hard NACK, this is completely out of scope, anyone in the target audience should either know what FTBFS is (prospective +1ers) or not care about that level of detail (people wanting to know what those damn reports are all about).